handbook/engineering/search/index.md

```diff
@@ -1,3 +1,29 @@
-# Search
+# Search team
 
-TODO: identify an owner to fill out this page and update CODEOWNERS
\ No newline at end of file
+## Vision
+
+Build and deliver Universal Code Search to all developers.
+
+## Contact
+
+- #search channel or @searchers on Slack.
+- [@sourcegraph/search](https://github.com/orgs/sourcegraph/teams/search) team or [search label](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+is%3Aopen+label%3Asearch) on GitHub.
```

Comment:
Can you please rename this label to be `team/search` on GitHub (to be consistent with other team labels) and then update the reference here